modules/articles/client/config/articles.client.routes.js

```diff
@@ -38,10 +46,13 @@
         controller: 'ArticlesController',
         controllerAs: 'vm',
         resolve: {
-          articleResolve: getArticle
+          articleResolve: getArticle,
+          pageTitle: ['articleResolve', function (articleResolve) {
+            return 'MEAN.JS - Edit Article ' + articleResolve.title;
```

Comment:
You'll have to put the state into it's own var, since `this` will change..
```javascript
function getArticle($stateParams, ArticleService) {
  var state = this;
  ......
    state.data.pageTitle += article.title;
  .....
```